lib/cesiumGeometryToGltfPrimitive.js

```diff
@@ -1,10 +1,89 @@
 'use strict';
+var findAccessorMinMax = require('./findAccessorMinMax');
 var writeAccessor = require('./writeAccessor');
+var mergeBuffers = require('./mergeBuffers');
+var uninterleaveAndPackBuffers = require('./uninterleaveAndPackBuffers');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+var WebGLConstants = Cesium.WebGLConstants;
+
 
 module.exports = cesiumGeometryToGltfPrimitive;
 
+function createAttributeSemantic(gltf, primitive, semantic, attributeLength) {
+    var type;
+    var bytesPerComponent;
+
+    switch (semantic.replace(/([0-9]|_)/g, '')) {
```

Comment:
This regex doesn't work for `TEXCOORD_0` (Also fix in `gltfPrimitiveToCesiumGeometry`).